From ffb2edba65e4f08d05f1346fbea8889a9fa8cb70 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 22 Sep 2016 13:31:59 -0500
Subject: [PATCH 02/35] Revert "drm/tilcdc: Take CRTC lock when calling
 tilcdc_crtc_disable()"

This reverts commit 91931688a4340b0bc6464549acf098010ca9a0ad.
---
 drivers/gpu/drm/tilcdc/tilcdc_crtc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/tilcdc/tilcdc_crtc.c b/drivers/gpu/drm/tilcdc/tilcdc_crtc.c
index af07320..6495831 100644
--- a/drivers/gpu/drm/tilcdc/tilcdc_crtc.c
+++ b/drivers/gpu/drm/tilcdc/tilcdc_crtc.c
@@ -230,9 +230,7 @@ static void tilcdc_crtc_destroy(struct drm_crtc *crtc)
 	struct tilcdc_crtc *tilcdc_crtc = to_tilcdc_crtc(crtc);
 	struct tilcdc_drm_private *priv = crtc->dev->dev_private;
 
-	drm_modeset_lock_crtc(crtc, NULL);
 	tilcdc_crtc_disable(crtc);
-	drm_modeset_unlock_crtc(crtc);
 
 	flush_workqueue(priv->wq);
 
-- 
2.9.3

